.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+name: build holochain
+on:
+  workflow_dispatch:
+  pull_request:
```

Comment:
```suggestion
```

Reply:
BTW, it requires only 2 minutes even if it has to re-build holochain.

Reply:
ah i see,that's probably when it doesn't have to rebuild the dependencies, right?